pkg/virt-api/webhooks/mutating-webhook/mutators/migration-create-mutator_test.go

```diff
@@ -17,81 +17,98 @@
  *
  */
 
-package mutators
+package mutators_test
```

Comment:
small nit: `mutatorstest` or `mutatorsTest`, This way of naming is in line with Go's standard.

Reply:
In a single package, one can have files of the package name - in this case `mutators` or their tests package with the `_test` suffix.

This was done in order to perform [black box testing](https://en.wikipedia.org/wiki/Black-box_testing) - test only the external API of the mutator.

Please see https://pkg.go.dev/testing and https://pkg.go.dev/cmd/go#hdr-Test_packages